docs/usage/HPC.md

```diff
@@ -0,0 +1,118 @@
+# High-Performance Computing (HPC)
+
+## Did you know?
+
+HippUnfold can automatically split its internal jobs and distribute them across a high-performance computing (HPC) cluster. This means you don’t have to rely only on your local workstation or manually batch subjects into separate job submissions. Instead, **Snakemake** handles job distribution, while HippUnfold provides sensible default resource settings for each step.
+
+This setup lets you parallelize and speed up processing while still respecting resource allocation constraints defined by your cluster.
+
+---
+
+## Before you start
+
+When running on a cluster, please check the following:
+
+* **Installation path**: Make sure HippUnfold is installed in a directory that is accessible from all compute nodes (e.g., shared network storage).
+
+* **Containers**: If you are using Apptainer/Singularity or Docker, confirm that the runtime is available on compute nodes.
```

Comment:
can you actually use hippunfold called from a container along with a executor plugin? I didn't know there was a way for the remote jobs to call the container properly.. Unless you've tested this we could remove this statement (or say you cannot use the hippunfold container with executor plugins)